operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -42,15 +47,36 @@ class TurbineLongTermGrossEnergy(object):
     """
 
     @logged_method_call
-    def __init__(self, plant):
+    def __init__(self, plant, UQ = 'Y', num_sim = 100, uncertainty_scada=0.005,
+                  uncertainty_wind_bin_thresh=(1, 3), uncertainty_max_power_filter=(0.8, 0.9), 
+                  uncertainty_correction_threshold=(0.85, 0.95)):
+    
         """
         Initialize turbine long-term gross energy analysis with data and parameters.
         Args:
          plant(:obj:`PlantData object`): PlantData object from which TurbineLongTermGrossEnergy should draw data.
-         max_power_filter(float): 
+         UQ:(:obj:`str`): choice whether to perform ('Y') or not ('N') uncertainty quantification
+         num_sim:(:obj:`int`): number of Monte Carlo simulations
+         uncertainty_scada(:obj:`float`): uncertainty imposed to scada data
+         uncertainty_max_power_filter(:obj:`float`): Maximum power threshold (fraction) to which the bin filter 
+                                            should be applied (default 0.85)
+         uncertainty_wind_bin_thresh(:obj:`float`): The filter threshold for each bin (default is 2 m/s)
+         uncertainty_correction_threshold(:obj:`float`): The threshold (fraction) above which daily scada energy data
+                                                should be corrected (default is 0.90)
```

Comment:
The last three arguments are tuples, but the docstring datatypes are only listed as floats.

Reply:
...and changed!